signer/tests/integration/postgres.rs

```diff
@@ -702,3 +706,147 @@ async fn writing_transactions_postgres() {
     assert_eq!(num_rows, count as usize);
     signer::testing::storage::drop_db(store).await;
 }
+
+/// Here we test that we can store completed deposit events.
+#[cfg_attr(not(feature = "integration-tests"), ignore)]
+#[tokio::test]
+async fn writing_completed_deposit_requests_postgres() {
+    let db_num = DATABASE_NUM.fetch_add(1, Ordering::SeqCst);
+    let store = signer::testing::storage::new_test_database(db_num).await;
+
+    let mut rng = rand::rngs::StdRng::seed_from_u64(51);
+    let event: CompletedDepositEvent = fake::Faker.fake_with_rng(&mut rng);
```

Comment:
Not something for this PR, but I would suggest moving from `fake` to [proptest](https://crates.io/crates/proptest) [strategies](https://altsysrq.github.io/proptest-book/proptest/tutorial/strategy-basics.html) and writing [proptests](https://altsysrq.github.io/proptest-book/proptest/tutorial/macro-proptest.html) instead. It provides much more robust testing using a multitude of different randomized values w/value shrinking etc. vs. the one random value you get here.

Reply:
Yeah, that's a good idea. I'll give it look.